service/controller/v2/context/error.go

```diff
@@ -0,0 +1,12 @@
+package context
+
+import (
+	"github.com/giantswarm/microerror"
+)
+
+var serviceNotFound = microerror.New("service not found")
```

Comment:
Also error there is `var notFoundError = microerror.New("not found")`:  https://github.com/giantswarm/aws-operator/blob/b80db2bdc3ea8533f4ede4f9df0a639ca1650214/service/controller/v14/controllercontext/error.go#L7